prottp.go

```diff
@@ -58,24 +57,24 @@ func NewError(statusCode int, body proto.Message) ErrorWithStatus {
 	return &respError{statusCode: statusCode, body: body}
 }
 
-func Handle(mux *http.ServeMux, service Service, mws ...server.Middleware) {
+func Handle(mux *http.ServeMux, service Service, interceptor grpc.UnaryServerInterceptor, mws ...server.Middleware) {
```

Comment:
This will break all existing calls to `Handle`? a `nil` interceptor will need to be added everywhere?